reactor-core/src/main/java/reactor/core/publisher/EmitterProcessor.java

```diff
@@ -51,7 +51,8 @@
  * @param <T> the input and output value type
  *
  * @author Stephane Maldini
- * @deprecated Prefer clear cut usage of {@link Sinks}, to be removed in 3.5
+ * @deprecated To be removed in 3.5. Prefer clear cut usage of {@link Sinks}:
+ * <pre>Sinks.many().multicast().onBackpressureBuffer();</pre> for example.
```

Comment:
rather than "for example": `variations of Sinks.many().multicast().onBackpressureBuffer()` (since the only other alternative IIRC is `onBackpressureError()`, which is the `DirectProcessor`)